### ISSUE TITLE
[A11y Audit]: Case Studies

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5467]
This audit is a buddy task, to be preformed soon after the standardization of the Case Study pages.

### Case Studies
- [Nutanix case study](https://about.sourcegraph.com/case-studies/nutanix-fixed-log4j-with-sourcegraph) - `/case-studies/nutanix-fixed-log4j-with-sourcegraph`
- [FactSet case study](https://about.sourcegraph.com/case-studies/factset-migrates-from-perforce-to-github) - `/case-studies/factset-migrates-from-perforce-to-github`
- [Cloudfare case study](https://about.sourcegraph.com/case-studies/cloudflare-accelerates-debugging-and-improves-security) - `/case-studies/cloudflare-accelerates-debugging-and-improves-security`
- [Indeed case study](https://about.sourcegraph.com/case-studies/indeed-accelerates-development-velocity) - `/case-studies/indeed-accelerates-development-velocity`
- [Workiva case study](https://about.sourcegraph.com/case-studies/workiva-automates-large-scale-code-changes) - `/case-studies/workiva-automates-large-scale-code-changes`
- [CERN case study](https://about.sourcegraph.com/case-studies/cern-reduces-technical-debt) - `/case-studies/cern-reduces-technical-debt`
- [Criteo case study](https://about.sourcegraph.com/case-studies/criteo-tackles-big-code) - `/case-studies/criteo-tackles-big-code`
- [f5 case study](https://about.sourcegraph.com/case-studies/f5-streamlines-collaboration-globally) - `/case-studies/f5-streamlines-collaboration-globally`
- [Convoy case study](https://about.sourcegraph.com/case-studies/convoy-improved-on-boarding) - `/case-studies/convoy-improved-on-boarding`
- [Lyft case study](https://about.sourcegraph.com/case-studies/lyft-monolith-to-microservices) - `/case-studies/lyft-monolith-to-microservices`
- [Quantcast case study](https://about.sourcegraph.com/case-studies/quantcast-large-scale-refactoring) - `/case-studies/quantcast-large-scale-refactoring`
- [SoFi case study](https://about.sourcegraph.com/case-studies/sofi-moves-fast-on-hundreds-of-microservices) - `/case-studies/sofi-moves-fast-on-hundreds-of-microservices`
- [THORN case study](https://about.sourcegraph.com/case-studies/we-are-thorn) - `/case-studies/we-are-thorn`

### User Journey:
Since our case studies all use the same layout, auditing one case study page should be sufficient. A cursory look over the other case study pages can be given to ensure all behavior is the same.

- Navigate to the [case study index](https://about.sourcegraph.com/case-studies).
- Click through CTA at bottom of page.
- Click a case study card to open.
- Click any given link on the case study.
- Click back to case studies from the Hero.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

Video example [here](https://www.loom.com/share/21a5d671fc2a44b6be9f61335389c456).

1. Navigate user journey using **only** the keyboard
2. Navigate user journey **without** looking at your screen, using the screen reader.
3. Navigate user journey using a viewport with a **width of 320px**.
4. Work through relevant sections from the detailed checklist and ensure there are no issues.
5. Run an [ARC Toolkit test](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-arc-toolkit) on the webpage for any changing state.
6. If the issue is trivial & you have time, feel free to address the bug now. Otherwise, raise any larger bugs using the [accessibility issue template](https://github.com/sourcegraph/about/issues/new?assignees=&labels=accessibility%2Cwcag%2F2.1%2Cwcag%2F2.1%2Ffixing&template=accessibility_issue.yaml&title=%5BAccessibility%5D%3A+).

**Note:** [The Front End Platform team](https://join.slack.com/share/enQtMzY1OTY3MDkxNTAwOS03ZmFlNzgwNWJhOGU3NjU4MDY1NzVkOTdiNjQwMWQyNjJmNjBkNzk1OWQ5ZDIxOTNjNmRhOWIyMzUwMzhmZjNh) is planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by st0nebraker** [comment]
Closed with 3 new issues created: #5630, #5628, #5627
